pkg/proxy/kafka.go

```diff
@@ -322,20 +346,17 @@ func handleRequests(done <-chan struct{}, pair *connectionPair, c *proxyConnecti
 		}
 
 		if err != nil {
-			if record != nil {
```

Comment:
Why was this removed? Shouldn't we be putting this in access log?

Reply:
record was always nil. This was dead code.